src/content/structured/get-started/migrating-to-v3.mdx

```diff
@@ -0,0 +1,91 @@
+---
+path: "/get-started/migrating-to-v3"
+
+navPriority: 4
+
+date: "2025-02-13"
+
+title: "Migrating to v3"
+
+subTitle: "Developer guidance on migration from v2 to v3 of the UI-Kit"
+
+contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/get-started/development/releases-versioning.mdx"
+---
+
+## Ic-theme and dark mode
+
+With v3's arrival comes the introduction of the `ic-theme` component. This is responsible for managing the theme of the UI Kit and providing a dark mode. Guidance on using the `ic-theme` package can be found in the [INSERT IC-THEME DOCUMENTATION](/theme).
+
+## Codemod
+
+Codemod is a script used to automate the process of migrating code from one version to another. It is a tool that can be used to make large-scale changes to codebases quickly and efficiently.It is the simplest and easiest way to migrate your codebase to the latest version of the UI Kit.
+
+### What is this used for
+
+Given a directory, it will scan over files and find any relevant ICDS components that require a change either by component name or props used.
+
+### How to run
+
+This package will be usable as an executable requiring a directory and optional test boolean argument to cover tests. Further information can be found in the codemod package [README](https://github.com/mi6/ic-ui-kit/blob/v3.0.0/main/packages/codemod/README.md).
+
+```console
+- npx @ukic/codemod <dir> <test>
+```
+
+## Breaking Changes - web-components
+
+### 1. Prop Renaming
+
+- The `appearance` prop has been replaced with the `theme` and `monochrome` props.
+- The `auto-focus` prop has been renamed to `autofocus`.
+- The `color` prop has been renamed to `brandColor`.
+- The `icThemeChange` event has been renamed to `icBrandChange`.
+
+### 2. Component Renaming
+
+- `ic-card` has been renamed to `ic-card-vertical`.
+- `ic-data-entity` has been renamed to `ic-data-list`.
+
+### 3. Removed Props/Slots/Events/Methods
+
+- Deprecated props, slots, events, and methods have been removed entirely.
+- The `maxLength` prop has been removed. Instead, use the `maxCharacters` prop and the `hideCharCount` prop to hide the character count visually.
+
+### 4. Class Name Changes
+
+- Class names on the host for most components have been updated to include the component name in the class name. For example, `disabled` has been updated to `ic-select-disabled`.
+
+### 5. Event Name Changes
+
+- The events `sideNavExpanded` and `topNavResized` are now called `icSideNavExpanded` and `icTopNavResized`.
+
+### 6. CSS Token Changes
+
+- Some existing CSS tokens have been renamed following the move towards primitive, semantic, and component tokens.
+
+### 7. Component Behavior Changes
+
+- The `ic-select` component now needs to be used to get the multi-select variant (use the `multiple` prop), instead of `ic-select-with-multi` from `canary-web-components`.
+- The `ic-toast` component now shows a dismiss button when the toast is hovered over or focused.
```

Comment:
```suggestion
- The `ic-toast` component now shows a dismiss button when the toast is hovered over or focused.
-`ic-multi-select` (`IcMultiSelect`) is no longer a standalone component. The multi-select functionality can now be achieved using the `ic-select` (`IcSelect`) component and its `multiple` prop.

```tsx
// v2
<ic-select-with-multi
  id="select-default"
  label="What are your favourite types of coffee?"
  multiple="true"
></ic-select-with-multi>

// v3
<ic-select
  id="select-default"
  label="What are your favourite types of coffee?"
  multiple="true"
></ic-select>
```
```